static-site/content/SARS-CoV-2-Datasets.yaml

```diff
@@ -728,6 +728,17 @@ datasets:
     org:
       name: CITRIC Center for Bioinformatics Aga Khan University Pakistan
       url: https://www.aku.edu/Pages/home.aspx
+      
+  - url: https://nextstrain.org/community/AKU-CITRIC-Center-for-Bioinformatics/ncov/Pakistan_Contextual
```

Comment:
This URL should be <https://nextstrain.org/community/AKU-CITRIC-Center-for-Bioinformatics/ncov/Pakistan/Contextual> (without the underscore).

I could make that minor change myself but am holding off because more broadly if your goal is to:

> display two trees on same nextstrain page by selecting tree from the dropdown menu on left.

then you'll want to make the two datasets (`Pakistan.json` and `Pakistan_Contextual.json` + their other associated files) at the same "level" by either:

1. Renaming `Pakistan.json` (and its associated files) to something like `Pakistan_Focused.json`.
2. Renaming `Pakistan_Contextual.json` (and its associated files) to something like `Pakistan-Contextual.json`.

With option (1) you'd end up with a dropdown menu allowing you to select between "Focused" and "Contextual". With (2) you'd end up selecting between "Pakistan" and "Pakistan-Contextual".

Let me know if I can clarify anything.

Reply:
Thank you very much @tsibley ! I got it. The second option suits best in my case. Now two trees can be viewed on single nextstrain web page. There is no need to merge changes

![contextual_tree1](https://user-images.githubusercontent.com/92725781/142416769-40d19ae0-a364-452b-95a6-a749445f7749.jpeg)

![contextual_tree2](https://user-images.githubusercontent.com/92725781/142416778-aa4a518c-6b5b-42d9-ad3d-59ed395c09e9.jpeg)

 Pakistan and related contextual samples tree is now displayed on nextstrain webpage

Thank you very much for your guidance! I really appreciate it .

Reply:
Excellent! Glad to be of help. I'll close this PR without merging.